SwiftMessageBar/Message.swift

```diff
@@ -60,6 +62,13 @@ final class Message: UIView {
     let titleLabel = initTitle()
     let messageLabel = initMessage()
     
+    if languageDirection == .rightToLeft {
+      titleLabel.textAlignment = .right
```

Comment:
Sure. I am giving explicit .right alignment because there is a scenario for one of my application where device language can be English but inside the application user selects Arabic so in that  case i think .natural will not work. But you can verify.